### ISSUE TITLE
shape tail in upheno file

### ISSUE THREAD

**Issue by sbello** [obophenotype/upheno#961]
I assume the phenotype 'shape tail' in the uPheno file comes from this pattern 

https://raw.githubusercontent.com/obophenotype/upheno/master/src/patterns/dosdp-dev/abnormalShapeOfAnatomicalEntityByShapeType.yaml

The way this has been entered we get a nonsensical phenotype term of 'shape tail' with a nonsensical definition 'Any structural anomaly that causes tail to be shape shaped.'

We should review and revise this pattern as needed. It maybe that general/grouping classes may need a variation on the standard label.

**Comment by sbello** [comment]
This is true of all the high level shape terms as a result of stripping the word 'abnormal' out.

**Comment by matentzn** [comment]
Fixed in the latest release: https://github.com/obophenotype/upheno-dev/releases/tag/2025-01-11

Was more work then expected as we were missing a pattern and it was hard to shift some EQs between conceptually overlapping patterns..
